ci-operator/config/openshift/installer/openshift-installer-master.yaml

```
@@ -236,6 +238,7 @@ tests:
     workflow: openshift-e2e-gcp-sharednetwork
 - as: e2e-azure
   optional: true
+  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(alibabacloud|aws|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
```

```suggestion
  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|terraform/providers/)/(alibabacloud|aws|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
```
1. let's not bother with images, very few changes are made in those dirs
2. let's include the new terraform provider dirs
can you make this change or equivalnet across the triggers?